README.md

```diff
@@ -42,9 +42,9 @@ The HRZ-Pool computers run Ubuntu Linux. Download the `Miniconda3 Linux 64-bit`
 - Open a terminal by pressing `Ctrl+Alt+T`. Navigate into this directory by typing `cd exercise_intro-yourname`. Type
 
   ```bash
-  pip install -r requirements.txt
```

Comment:
@v0lta it's not so great to have nox actually be in the requirements for the package. this is a tool that should be on your computer, just like `pip` (which you also don't put into the requirements)